docs/docs/reference/built-in-components/gatsby-slice.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Gatsby Slice API
+---
+
+> Support for the Gatsby Slice API was added in `gatsby@5.0.0`.
+
+To further the improvements seen by [Incremental Builds](https://www.gatsbyjs.com/blog/2020-04-22-announcing-incremental-builds/), Gatsby includes a built-in `<Slice>` component to divide pages into incrementally built "slices".
+
+By using a new `<Slice>` React component in combination with the [`createSlice`](/docs/reference/config-files/actions/#createNodeField) API for common UI features, Gatsby will be able to build and deploy individual pieces of your site that had content changes, not just entire pages.
+
+## `<Slice>` enables faster builds in Gatsby Cloud
+
+With the introduction of [Incremental Builds](https://www.gatsbyjs.com/blog/2020-04-22-announcing-incremental-builds/), Gatsby Cloud has been able to reduce build times signficantly by only building the pages that changed. The `<Slice>` component helps reduce those builds times further.
+
+Common components that are shared across the majority of pages on your site might include a navigation bar, footer, or contact form. In today's frameworks, when you re-order the navigation bar items, the entire site needs to be rebuilt. However, if the navigation bar was created as a Gatsby Slice, the new navigation items only need to built once and all pages will pull the new navigation bar when it's needed.
+
+## How to use Gatsby's `<Slice>`
+
+For this example, let's use the same scenario as described above - a large site that has a shared navigation bar.
+
+```typescript:title=src/layouts/default-layout.tsx
+import { NavigationBar, Footer } from "../components"
+
+export const DefaultLayout = ({ children, navigationBarClassName }: DefaultLayoutProps) => {
+  /// ...
```

Comment:
Adding those `/// ...` is adding extra noise, we should remove them in all code examples